pandas/tests/util/test_util.py

```diff
@@ -467,3 +467,11 @@ def test_set_locale(self):
 
         current_locale = locale.getlocale()
         assert current_locale == self.current_locale
+
+
+def test_make_signature():
+    # See GH 17608
+    _ = make_signature(validate_kwargs)
```

Comment:
you need to assert something here

Reply:
This test will fail under the status quo.